website/versioned_docs/version-3.x/tab-based-navigation.md

```diff
@@ -85,14 +88,61 @@ export default createBottomTabNavigator(
 );
 ```
 
-<a href="https://snack.expo.io/@react-navigation/tabs-with-icons-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
+<a href="https://snack.expo.io/@sunnylqm/tabs-with-icons-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
 
 Let's dissect this:
 
 * `tabBarIcon` is a property on `navigationOptions`, so we know we can use it on our screen components, but in this case chose to put it in the `createBottomTabNavigator` configuration in order to centralize the icon configuration for convenience.
 * `tabBarIcon` is a function that is given the `focused` state, `tintColor`, and `horizontal` param, which is a boolean. If you take a peek further down in the configuration you will see `tabBarOptions` and `activeTintColor` and `inactiveTintColor`. These default to the the iOS platform defaults, but you can change them here. The `tintColor` that is passed through to the `tabBarIcon` is either the active or inactive one, depending on the `focused` state (focused is active). The orientation state `horizontal` is `true` when the device is in landscape, otherwise is `false` for portrait.
 * Read the [full API reference](bottom-tab-navigator.html) for further information on `createBottomTabNavigator` configuration options.
 
+## Add badges to icons
+
+Sometimes we want to add badges to some icons. A common way is to use an extra view container and style the badge element with absolute positioning.
+
+```js
+export default class IconWithBadge extends React.Component {
+  render() {
+    const { name, badgeCount, color, size } = this.props;
+    return (
+      <View style={{ width: 24, height: 24, margin: 5 }}>
+        <Ionicons name={name} size={size} color={color} />
+        { badgeCount > 0 && (
+          <View style={{
+            // Absolute positioning ref: https://developer.mozilla.org/en-US/docs/Web/CSS/position#Absolute_positioning
```

Comment:
can we condense this comment to one line? maybe just `// If you're using react-native < 0.57 overflow outside of parent will not work on Android, see https://git.io/fhLJ8`

Reply:
I updated the comment in https://snack.expo.io/@react-navigation/tabs-with-icons-v3 let me know if that works for you

Reply:
Sure. Updated.